changelog/2016.bugfix.rst

```diff
@@ -0,0 +1 @@
+Updated the doc string notes section, edited typos in text.
```

Comment:
```suggestion
Updated the docstring of `~plasmapy.dispersion.numerical.kinetic_alfven_.kinetic_alfven`.
```
Including the link so that someone reading the changelog will know which docstring was changed.